src/components/Match/matchColumns.jsx

```diff
@@ -128,7 +128,7 @@ export default (strings) => {
         displayFn: (row, col, field) => field || '-',
         sortFn: true,
         sumFn: true,
-        color: constants.colorGreen,
+        color: 'hsla(123, 25%, 57%, 1)',
```

Comment:
why hsla instsead of rgba or hex?

Reply:
hsla allows to adjust saturation values more easily.